projects/igniteui-angular-elements/src/app/wrapper/wrapper.component.ts

```diff
@@ -14,6 +14,7 @@ type TemplateFunction = (arg: any) => TemplateResult;
 export class TemplateWrapperComponent {
 
     public templateFunctions: TemplateFunction[] = [];
+    public templateRendered = new EventEmitter<any>();
```

Comment:
The correct type would be:
```suggestion
    public templateRendered = new EventEmitter<HTMLElement>();
```
and the typing adjustments following this

Reply:
Also subject, cuz not an Angular component so no benefit, just overhead.